lib/active_model/serializer.rb

```diff
@@ -113,6 +113,56 @@ def initialize(object, options = {})
       end
     end
 
+    # @return [Hash] containing the attributes and first level
```

Comment:
Arguably these two methods should now be in one of the modules.. or in their own 'serializable' module..